content/en/tracing/serverless_functions/_index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Trace Serverless Functions
+kind: documentation
+description: 'Send traces from your Serverless Functions to Datadog'
+further_reading:
+    - link: 'serverless'
+      text: 'Serverless Monitoring with Datadog'
+---
+{{< img src="tracing/serverless_functions/ServerlessDistributedTrace.png" alt="Trace Serverless Functions"  style="width:100%;">}}
+
+In hybrid architectures, Serverless functions can form an important core of requests your application processes. Including these functions in your Distributed Traces can be critical for detecting performance bottlenecks and service outages in your distributed systems.
+
+## Choose your Tracing Library
+
+Depending on your language and configuration, choose between using the Datadog Lambda Library or AWS X-Ray for your traces.
+
+{{< img src="tracing/serverless_functions/LambdaLibraryXRay.png" alt="Lambda Library or AWS X-Ray"  style="width:100%;">}}
+
+**Note:** If you are tracing across lambda functions and hosts, you will need to use **both** Datadog APM and AWS X-Ray.
+
+## Enable Datadog APM
+
+Instructions for enabling Datadog APM on your lambda functions are contained within the [Serverless][1] section of our documentation.  You will also need to [install the AWS integration][2].
+
+## Enable AWS X-Ray
+
+Instructions for enabling AWS X-Ray on your lambda functions are contained within the [AWS X-Ray Tracing][3] section of our documentation.  You will also need to [install the AWS integration][2].
+
+## Augment AWS X-Ray Tracing with Datadog APM
+
+{{< img src="integrations/amazon_lambda/lambda_tracing.png" alt="architecture diagram for tracing AWS Lambda with Datadog" >}}
+
+There are some scenarios in which AWS X-Ray Tracing will be used side-by-side with Datadog APM.  Note that this will have an impact on your bill, and if you are unsure whether to use Datadog APM or AWS X-Ray, please reach out to [our support team][4] to discuss, or read about the [differences](#choose-your-Tracing-Library)
```

Comment:
```suggestion
There are some scenarios in which AWS X-Ray Tracing will be used side-by-side with Datadog APM.  Note that this will have an impact on your bill, and if you are unsure whether to use Datadog APM or AWS X-Ray, please reach out to [our support team][4] to discuss, or read about the [differences](#choose-your-Tracing-Library).
```